pkg/monitoring/rules/alerts/prometheus.go

```diff
@@ -0,0 +1,57 @@
+package alerts
+
+// This package makes sure that the PrometheusRule is present with the right configurations.
+// This code was taken out of the operator package, because the operand reconciliation is done
+// only if the HyperConverged CR is present. But we need the alert in place even if the CR was
+// not created.
```

Comment:
is this comment needed? if you think so, I think it should at least appear before package declaration

Reply:
sorry it's leftovers. and I will fix it in alerts.go to be before the pkg name